spec/pdfh/utils/rename_validator_spec.rb

```diff
@@ -3,7 +3,7 @@
 RSpec.describe Pdfh::RenameValidator do
   subject(:validator) { described_class.new(name_template) }
 
-  let(:name_template) { "document {original}-{period}-{year}-{month}-{type}-{subtype}-{extra}" }
+  let(:name_template) { "document {original}-{period}-{YEAR}-{month}-{type}-{subtype}-{extra}" }
```

Comment:
In the readme, you changed YEAR from uppercases to lowercases, and here you changed YEAR from lowercases to uppercases. And it's the only one with that format.  Is there a reason for that?